### PR TITLE
Preparing build to include localized resources

### PR DESCRIPTION
**Pull request by Benglin**
### Purpose

This pull request updates `DynamoCore.csproj` to copy all localized resources from `extern/Localized` folder to `%OutputPath% folder during build. These binaries that now reside in %OutputPath% will be consumed by installer build to produce MUI installer.

The `DynamoSandbox.exe` has also been updated to take in a new *locale* related command line argument. For example, to launch Dynamo in Japan locale, do the following:

    DynamoSandbox.exe /L "ja-JP"

### Declarations

Check these iff you believe they are true

- [ ] The code base is in a better state after this PR
- [ ] The level of testing this PR includes is appropriate

### Reviewers

Hi @nguyen-binh-minh, this is the work that we have talked about before. Could you see if anything in the installer step (both `*.msi` and `*.exe` installers) is missing?

I have tried the `exe` installer and it correctly packages and installs all localized resources. As part of your review, can you help to try out `*.msi` installer on `Benglin:LocalizedBuild` branch (not this branch)? That branch has all the necessary localized resources. Thanks!

### FYIs

@RodRecker, this is the work that will enable us to build MUI installers, but the actual localized resources are incomplete (e.g. `libg_locale` folder has gone missing for a week now and `gallery` is a new folder for localization), therefore the daily builds will have incomplete localized resources contained in them. Once WWL team delivers a new round of localized resources, they will automatically be completing our installers.

### CODE REVIEW

File: tools/install/DynamoInstaller.iss [hunk 1]
```diff
@@ -199,7 +199,7 @@ begin
     begin
 	  MsgBox('Dynamo requires an installation of Revit 2014, Revit 2015 or Revit 2016 in order to proceed!', mbCriticalError, MB_OK);
       result := false;
-    end
+    end;
```
Comment (Benglin) [inline tools/install/DynamoInstaller.iss]: The latest `InnoSetup` version on my machine refused to parse this script without the semi-colon here.

**Comment by RodRecker** [comment]
@Benglin This looks good. What happens when the localized resources are not complete? Will the EN ones be used?

**Comment by Benglin** [comment]
Thanks for checking @RodRecker, yes, for both `libg_locale` and `gallery`, there are built-in logic to fall back to `en-US` resources should they go missing. I'll see what is the ETA for WWL team to drop us these missing resources and then decide if this should go in first.

**Comment by nguyen-binh-minh** [comment]
@Benglin LGTM

**Comment by Benglin** [comment]
Hi @nguyen-binh-minh, as discussed I have also updated MSI installer script, PTAL. Thanks.

**Comment by nguyen-binh-minh** [comment]
@Benglin LGTM

File: src/DynamoSandbox/Program.cs [hunk 2]
```diff
@@ -78,6 +80,76 @@ public string CommonDataRootFolder
...
+                CommandFilePath = commandFilePath,
+                StartUpDynFilePath = startUpDynFilePath
+            };
+        }
+
+        internal string Locale { get; set; }
+        internal string CommandFilePath { get; set; }
+        internal string StartUpDynFilePath { get; set; }
+    }
+
```
Comment (Benglin) [inline src/DynamoSandbox/Program.cs]: This whole chunk is taken from `Dynamo Studio`.

**Comment by Benglin** [comment]
Thanks for reviewing, @nguyen-binh-minh!

**Pull request merged**